### PR TITLE
Course Roster : link table with courses and students is running

### PR DESCRIPTION
**Pull request by mlworle**
Having trouble iterating over course -- then students. Same problem with course loads in the index view.

spock@spock-HP-Pavilion-dm1-Notebook-PC:~/rails/Attinderance/app/views/course_rosters$ cat index.html.erb 
<h1> Attinderance | Courseload Index </h1>
<p>This is where a list of all courserosters will be found. </p>
<br>
<p>Course</p>
    <% @course_rosters.each do |courseroster| %>
        <ul>
        <%= courseroster.course.name%>
          <ul>
          <li><%= courseroster.student.name %></li>
          </ul>
        </ul>
    <% end %>

### CODE REVIEW

**Comment by edwardloveall** [comment]
You'll need to make this an erb output tag: `<%=`

**Comment by mlworle** [comment]
Played around a bit. Not sure what you meant.

**Comment by edwardloveall** [comment]
Oh you know what, I was wrong. Ignore that. Let me take a closer look.

File: app/controllers/course_roster_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class CourseRosterController < ApplicationController
```
Comment (edwardloveall) [inline app/controllers/course_roster_controller.rb]: The name should be plural: `CourseRostersController` and same with the file: `course_rosters_controller.rb`

Perhaps you ran a generator wrong?

**Comment by mlworle** [comment]
That is true. Fixed it but didn't push the corrections. Sorry.

**Comment by edwardloveall** [comment]
:+1:

**Comment by mlworle** [comment]
Merge?

**Comment by edwardloveall** [comment]
Yup!

**Pull request merged**